client/my-sites/checkout/checkout-thank-you/marketplace/use-atomic-transfer.tsx

```diff
@@ -32,6 +33,8 @@ export function useAtomicTransfer(
 	const [ isAtomicTransferCheckComplete, setIsAtomicTransferCheckComplete ] = useState(
 		! isAtomicNeeded
 	);
+	const previousIsAtomicNeeded = usePrevious( isAtomicNeeded );
+	const isAtomicNeededTransition = ! previousIsAtomicNeeded && isAtomicNeeded;
```

Comment:
I would indicate in the variable name that is the specific transition from `false` to `true`. So I would use a name it `isAtomicNeededActivated` or similar.

Reply:
updated here https://github.com/Automattic/wp-calypso/pull/82192/commits/58e8f91b2e2bd818afc0a051b6be4839cb159426